features/partners.feature

```diff
@@ -50,12 +48,12 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
       | naam                      | waarde   |
       | datum aangaan (06.10)     | 20040506 |
       | soort verbintenis (15.10) | P        |
-      | indicatie onjuist (84.10) |          |
+      En de partner heeft GEEN ontbindingHuwelijkPartnerschap gegevens
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 999993380                       |
-      | fields              | partners.naam.voornamen         |
+      | fields              | partners                        |
```

Comment:
Hier geldt ook dezelfde opmerking van scenario 'Actueel huwelijk'. Is het niet relevant/nodige om te valideren dat de aangaanHuwelijkPartnerschap gegevens van de GBA API worden overgenomen en getransformeerd? Dit wordt namelijk niet in andere partners scenario's gevalideerd.

Reply:
ik breidt fields uit tot specifieke velden die gecontroleerd worden